src/eipmi_decoder.erl

```diff
@@ -183,9 +189,9 @@ session(<<?EIPMI_RESERVED:4, 6:4, E:1, A:1, P:6, I:32/little, S:32/little, L:16/
               1 -> outbound_auth_seq_nr
           end,
     {[{auth_type, rmcp_plus},
-      {encrypted, eipmi_utils:get_bool(E)},
```

Comment:
Sorry for messing that up 🤦

Reply:
Oh! :sweat_smile: I had assumed I did it.